### DISCUSSION TITLE
If the language is Japanese, Door have the same label for Wiring interface.

### DISCUSSION THREAD

**Discussion by kaedepon** [FakeFishGames/Barotrauma#13739] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

It is the wiring interface for the door.
In English, there are written as "STATE_OUT" and "CONDITION_OUT".
In Japanese, both are written as "状態出力", which is confusing when wiring.
*For reference, the hatch is also translated the same way, Probably others too.
# In English
![door_en](https://github.com/FakeFishGames/Barotrauma/assets/47914478/53ff6f84-0721-4df5-b364-3ab4fd6f78c5)
# In Japanese
![door_jp](https://github.com/FakeFishGames/Barotrauma/assets/47914478/b23140c9-ee04-4a33-bd3b-d7e83dffda69)

----
I think the reason for this problem is that the literal translation of "State" and "Condition" into Japanese would both be "状態".
In this case, I think the Japanese will understand this translation.
 - "STATE_OUT" = "状態出力"
 - "CONDITION_OUT" = "耐久値出力"

### Reproduction steps

1. Select Japanese Language.
2. Interact the door with a screwdriver.
3. Watch the labels on the wiring interface.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.3.0.2

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

We've fixed this issue at our end now, the fix should be coming up in the next update.
